app/helpers/formats_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module FormatsHelper
+  def date_format(date)
```

Comment:
`date_format`....which format? There might be dozens of date formats 🤔 
Having a more explicit name would be better, like `short_date` or making your method to required the desired format, but this doesn't make sense, method would be bigger that what it does haha